addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -2598,7 +2598,7 @@ X[]
                     await testEditor(BasicEditor, {
                         contentBefore: '<p>abc[] </p>',
                         stepFunction: insertParagraphBreak,
-                        contentAfter: '<p>abc</p><p>[]<br></p>',
+                        contentAfter: '<p>abc</p><p>[]</p>',
```

Comment:
I was just taking a quick look and saw this and thought I'd already comment here because this is incorrect. The `<br>` is needed. Without it, the `<p>` will have 0 height and the cursor can't be put inside it.